Test/git-issues/git-issue-2959.dfy.expect

```diff
@@ -0,0 +1,5 @@
+git-issue-2959.dfy(14,5): Error: variable 'b', which is subject to definite-assignment rules, might be uninitialized here
+git-issue-2959.dfy(27,7): Error: variable 'y', which is subject to definite-assignment rules, might be uninitialized here
+git-issue-2959.dfy(31,11): Error: unless an initializer is provided for the array elements, a new array of 'int' must have empty size
```

Comment:
They do verify, but they don't pass the determinism test, but that stage was hidden by the verification errors in the other methods. So I split the tests apart so that all errors are reported. Good spotting.